Survey Checks/README.md

```diff
@@ -0,0 +1,32 @@
+# Survey checks
+
+Here we provide detailed instructions for preparing your electronic survey with the aim of adding survey checks and creating a real-time high-frequency check dashboard.
```

Comment:
The iesurveychecks toolkit details the full workflow to create a dashboard for real-time checks of survey data. This toolkit uses SurveyCTO software for data collection and google sheets for dashboard display.

Reply:
This gets at a comment I made elsewhere - it should be clearer that this toolkit is focused on the implementation of the dashboard (not survey programming itself)